packages/cli-command/src/command.js

```diff
@@ -174,4 +174,10 @@ export function command(name, definition, callback) {
   return runner;
 }
 
+// App Percy currently only supports `app:exec` command and its sub commands,
+// All the other commands are deemed for Percy Web.
+export function getExecType(command) {
+  return command.name?.includes('app') ? 'app' : 'web';
+}
+
```

Comment:
cli-command is so far decoupled from the actual commands. Here we are coupling it by including command specific logic in cli-command. 

I would recommend having `execType` as an instance variable here, which will be passed by original command.

Like in ` runCommandWithContext` you can add exectype as required arg.

Idea being to keep this decoupled, it should not know existence of app/web commands.

Reply:
we have decoupled it and added projectType property in percy.js